akd/src/directory.rs

```diff
@@ -306,6 +313,48 @@ impl<S: Storage + Sync + Send> Directory<S> {
     /// or a version that we are retiring.
     pub(crate) fn get_nodelabel<H: Hasher>(uname: &AkdKey, stale: bool, version: u64) -> NodeLabel {
         // this function will need to read the VRF key using some function
+        // Initialization of VRF context by providing a curve
+        let mut vrf = ECVRF::from_suite(CipherSuite::SECP256K1_SHA256_TAI).unwrap();
+        // Inputs: Secret Key, Public Key (derived) & Message
+        let secret_key =
+            hex::decode("c9afa9d845ba75166b5c215767b1d6934e50c3db36e89b127b8a622b120f6721")
```

Comment:
We should probably create a trait which allows getting & settings the public key of the VRF. That way it can be outsourced to the implementing library down-the-road at some point. We can have a test-implementation which is just a memory location, but the trait should be like the storage layer & injectable.

Additionally, if there is any hard-coded values (i.e. key values), they should map to a constant, so we don't suffer from copy-paste risks.